spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/DataFlowITProperties.java

```diff
@@ -21,78 +21,106 @@
 /**
  * @author Christian Tzolov
  */
-@ConfigurationProperties(prefix = "test.docker.compose")
+@ConfigurationProperties(prefix = "test")
 public class DataFlowITProperties {
 
-	/**
-	 * Default url to connect to dataflow
-	 */
-	private String dataflowServerUrl = "http://localhost:9393";
-
-	/**
-	 * default - local platform (e.g. docker-compose)
-	 * cf - Cloud Foundry platform, configured in docker-compose-cf.yml
-	 * k8s - GKE/Kubernetes platform, configured via docker-compose-k8s.yml.
-	 */
-	private String platformName = "default";
-
-	/**
-	 * Default url to connect to SCDF's Prometheus TSDB
-	 */
-	private String prometheusUrl = "http://localhost:9090";
-
-	/**
-	 * Default url to connect to SCDF's Influx TSDB
-	 */
-	private String influxUrl = "http://localhost:8086";
-
-	/**
-	 * Kubernetes tests require nginx-ingress and watchdog mechanism to expose the apps  URL
-	 * to public access. Later requires a preconfigured ingress servers domain name suffix.
-	 *
-	 * For example the Hydra AT environment hash the 'hydra.springapps.io' host name and app Urls
-	 * will have the form  https://partitioning-test-log-v1.hydra.springapps.io
-	 * or  https://partitioning-test-log-v1-1.hydra.springapps.io for multiple instance.
-	 */
-	private String kubernetesAppHostSuffix = "";
-
-	public String getDataflowServerUrl() {
-		return dataflowServerUrl;
-	}
-
-	public void setDataflowServerUrl(String dataflowServerUrl) {
-		this.dataflowServerUrl = dataflowServerUrl;
-	}
-
-	public String getPlatformName() {
-		return platformName;
-	}
+	private DockerProperties docker = new DockerProperties();
```

Comment:
I think we should find a better name Then `DockerProperties` as those properties has noting to do with docker or docker-compose. They are just IT/AT configuration properties.
the `docker.compose` suffix is unfortunate properties naming inherited from the initial DataFlowIT implementation.